1 table.sql

```diff
@@ -0,0 +1,15 @@
+use ArmyDB
+go
+drop table if exists Soldier
+go
+create table dbo.Soldier(
+	SoldierFirstName varchar(30) not null constraint c_Soldier_first_name_cannot_be_blank check(SoldierFirstName<>''),
+	SoldierLastName varchar(30) not null constraint c_Soldier_last_name_cannot_be_blank check(SoldierLastName<>''),
+	SSN char(10) not null,
+	DOB date not null constraint ck_Army_Soldier_DOB_must_be_at_least_17_years_ago check(datediff(year,DOB,getdate()) >= 17),
+	Residence varchar(25) not null constraint c_Soldier_Residence_cannot_be_blank check(Residence<>''),
+	EnlistmentDate date,
```

Comment:
EnlistmentDate should be at the age of 17+ and cannot be in future date.

Reply:
I had a constraint that should do this in DOB column